DESCRIPTION

```diff
@@ -12,3 +12,10 @@ LazyData: true
 Suggests: 
     testthat (>= 3.0.0)
 Config/testthat/edition: 3
+RoxygenNote: 7.2.1
+Imports: 
+    dplyr,
+    magrittr,
+    purrr,
+    readr,
+    tidyr
```

Comment:
For writing packages, is it better to call these out individually instead of the blanket `library(tidyverse)`?

Reply:
individually